game/plugin/cmd/src/teleport-cmd.plugin.kts

```diff
@@ -75,6 +75,30 @@ on_command("tele", PrivilegeLevel.ADMINISTRATOR)
         }
     }
 
+/**
+ * Teleports the player to another player.
+ */
+on_command("teleto", PrivilegeLevel.ADMINISTRATOR)
+    .then { player ->
+        val invalidSyntax = "Invalid syntax - ::teleto [player name]"
+
+        if (arguments.size == 1) {
+            val playerName = arguments[0]
+            val foundPlayer = player.world.getPlayer(playerName)
+
+            if(foundPlayer == null) {

```

Comment:
Can you reformat this file? Basically just need a space after the `if` here:
```kotlin
if(foundPlayer == null) {
```

Soon (tm) the linter will pick this up and let you know when you run `gradle check`

Reply:
Sounds good. Addressed in latest commit.